core/domain/stats_services.py

```diff
@@ -169,6 +169,17 @@ def get_versions_for_exploration_stats(exploration_id):
         exploration_id)
 
 
+def get_total_unresolved_answers_for_exploration(exploration_id):
+    """Returns the sum total of unresolved answers (across all states) of an
+    exploration."""
+    exploration = exp_services.get_exploration_by_id(exploration_id)
+    state_names = exploration.states.keys()
+
+    return sum(stats_domain.StateRuleAnswerLog.get(
```

Comment:
I agree it can be heavy on the system. Also, I think that the current `get_multi` method inside `StateRuleAnswerLog` class (in core.domain.stats_domain) will not will not help in this case. Can you please elaborate what did you meant by batching things. ?